README.md

```diff
@@ -72,6 +72,19 @@ Thermostat functions are only available as of Fritz!OS 6.36
 
 `getGuestWlan` returns a structure containing all wifi settings found in the Fritz!Box UI. The `setGuestWlan` function accepts either a settings structure such as this or a single boolean value.
 
+### LED status functions
+
+- Get the current LED status `getLEDStatus`
+- Set the LED status `setLEDStatus`
+
+**Note**
+
+`getLEDStatus` returns an object containing the String typed `led_display` property with following possible values, which can also  be passed as the `status` parameter to `setLEDStatus`:
```

Comment:
... with `led_display` containing the led status ...

I'd suggest cleaning up the API into receiving/returning a simple numeric value and performing object creation internally in the library.